src/Container.php

```diff
@@ -22,19 +22,67 @@ public function version()
     }
 
     /**
-     * {@inheritdoc}
+     * Get the base path of the Laravel installation.
+     *
+     * @param  string  $path
+     * @return string
      */
-    public function basePath()
+    public function basePath($path = '')
```

Comment:
Weird. Looks like the contract/interface is different from the concrete implementation.

Reply:
Yeah it’s wired but laravel/framework does the same thing 🤷‍♂️